ports/butterflysoup/Butterfly Soup.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
+# PORTMASTER: butterflysoup.zip, Butterfly Soup.sh
+
+# Prelude
+XDG_DATA_HOME=${XDG_DATA_HOME:-$HOME/.local/share}
+
+if [ -d "/opt/system/Tools/PortMaster/" ]; then
+  controlfolder="/opt/system/Tools/PortMaster"
+elif [ -d "/opt/tools/PortMaster/" ]; then
+  controlfolder="/opt/tools/PortMaster"
+elif [ -d "$XDG_DATA_HOME/PortMaster/" ]; then
+  controlfolder="$XDG_DATA_HOME/PortMaster"
+else
+  controlfolder="/roms/ports/PortMaster"
+fi
+
+source $controlfolder/control.txt
+[ -f "${controlfolder}/mod_${CFW_NAME}.txt" ] && source "${controlfolder}/mod_${CFW_NAME}.txt"
+get_controls
+
+# Variables 
+PORTEXEC="renpy/startRENPY"
+GAMEDIR="/$directory/ports/butterflysoup"
+RUNTIME="renpy_8.1.3"
+RENPYDIR="$GAMEDIR/renpy/"
```

Comment:
This should be without a slash at the end, it is causing double slashes in paths:
```suggestion
RENPYDIR="$GAMEDIR/renpy"
```